php/WP_CLI/Dispatcher/CompositeCommand.php

```diff
@@ -122,6 +125,15 @@ public function get_shortdesc() {
 		return $this->shortdesc;
 	}
 
+	/**
+	 * Get the hook name for this composite
+	 * command.
+	 *
+	 * @return string
+	 */
+	public function get_hook() {
+		return $this->hook;
+	}
 	/**
```

Comment:
Somehow this went from 2 empty lines to 0

```suggestion
	}

	/**
```